index.js

```diff
@@ -36,6 +36,10 @@ module.exports = function (opts) {
 		jasmine.loadConfig(opts.config);
 	}
 
+	if (opts.errorOnFail === undefined) {
+		opts.errorOnFail = true;
```

Comment:
This will modify the user supplied object, which you should never do. Use https://github.com/sindresorhus/object-assign at the top instead to copy the object and set a default value.